appjs/logicjavascript.js

```diff
@@ -3394,26 +3394,20 @@ function impsefind20() {
     var series = n * (Math.pow(n, 2) + 3 * n + 5) / 3;
     document.getElementById("impseans20").innerHTML = series;
 }
-function calceximpse20find(){
-    var n = document.getElementById("impse20");
-    n.value = 3;
-
-    impse20find();
-}
-function impse20find() {
-    var n = parseInt(document.getElementById("impse20").value)
-    if (!isNaN(n)) {
-        var series = n * (Math.pow(n, 2) + 3 * n + 5) / 3;
-        document.getElementById("impse20exp").innerHTML = "\\[It is basically a function with domain and co-domain as natural numbers and 0. \\]"
-        document.getElementById("impse20exp").innerHTML = "\\[if n > 0 and the number is not already included in the sequence,\\]"
-        document.getElementById("impse20exp").innerHTML = "\\[ a(n) = a(n - 1) - n \\]"
-        document.getElementById("impse20exp").innerHTML = "\\[else a(n) = a(n-1) + n. \\]"
-        document.getElementById("impse20ans").innerHTML = series;
-        renderMathInElement(document.getElementById("impse20exp"))
-    }
-    else
-        document.getElementById("impse20ans").innerHTML = "please enter valid input ";
-}
```

Comment:
i didnt delete it , i just commented it because there are 2 functions with name - impse20find

in one function , the program for  calculator was present 
in 2nd function , the working steps for it were present 

so i just merged them both (commented the 2nd one)